### PR TITLE
Add documentation for admin

### PR DESCRIPTION
**Pull request by clytemnestra**
### Subject

 Added documentation about how to remove Admin, maybe add more info about it in the future. We need an explanation about the admin interface and how to remove it, if people don't want additional fields in their admin dashboard.

### To do

- [ ] wait for opinions
- [ ] squash commits

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Please rename your PR title and commit comment according to the subject.

**Comment by clytemnestra** [comment]
Hm, I suck at rebasing.

File: Resources/doc/reference/admin.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Admin
+=====
+
+The bundle comes with an Admin interface for medias and galleries, that filters them by contexts.
+
+
+Removing the Admin 
+------------------
+
+There may be cases where you don't want the default admins to appear on your dashboard. For that, you need to set up a compiler pass that tags the services with ``show_in_dashboard = false``:
```
Comment (greg0ire) [inline Resources/doc/reference/admin.rst]: Please linebreak long lines.

File: Resources/doc/reference/admin.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+              $tags['sonata.admin'][0]['show_in_dashboard'] = false;
+              $definition->setTags($tags);
+  
+          }
+      }
+  }
+
+
+
+This will keep the services in the container. We need the media service because it is used in the `media widget <../../views/Form/media_widgets.html.twig>`_   to generate an edit link of the media through ``adminByAdminCode('sonata.media.admin.media')`` whenever you have a media in a form in your admin dashboard. If you don't want the admin routes to be accessible, either, then you will have to remove the service definitions from the container, and override the media_widgets template and remove the line that generates the url.
```
Comment (greg0ire) [inline Resources/doc/reference/admin.rst]: - "accessible either," (just one comma)
- please quote `media_widgets` with double backticks

**Comment by clytemnestra** [comment]
Phew, that should do!

**Comment by core23** [comment]
Don't merge this right now! 

Please have a look at https://github.com/sonata-project/SonataClassificationBundle/pull/221

**Comment by core23** [comment]
IMO, we don't need any special information for this rare case.

You could also configure your admins in the [configuration](https://sonata-project.org/bundles/admin/3-x/doc/reference/dashboard.html#using-the-config-yml).

**Pull request closed without merging**